homeassistant/components/bluecurrent/translations/nl.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Please do not include foreign translations. Translations can be managed via the Lokalise platform after this PR has been merged. For now, please remove the `nl.json` file.

Reply:
Removed translations.